src/test/java/com/google/api/codegen/testdata/php_main_library.baseline

```diff
@@ -42,20 +42,16 @@ use google\example\library\v1\GetBookRequest;
 use google\example\library\v1\GetShelfRequest;
 use google\example\library\v1\LibraryServiceClient;
 use google\example\library\v1\ListBooksRequest;
-use google\example\library\v1\ListBooksResponse;
 use google\example\library\v1\ListShelvesRequest;
-use google\example\library\v1\ListShelvesResponse;
 use google\example\library\v1\ListStringsRequest;
-use google\example\library\v1\ListStringsResponse;
 use google\example\library\v1\MergeShelvesRequest;
 use google\example\library\v1\MoveBookRequest;
 use google\example\library\v1\PublishSeriesRequest;
-use google\example\library\v1\PublishSeriesResponse;
 use google\example\library\v1\Shelf;
 use google\example\library\v1\SomeMessage;
 use google\example\library\v1\StringBuilder;
 use google\example\library\v1\UpdateBookIndexRequest;
-use google\example\library\v1\UpdateBookIndexRequest\IndexMapEntry;
+use google\example\library\v1\UpdateBookIndexRequest.IndexMapEntry;
```

Comment:
I ignored this when I saw in the first time because I wasn't sure why it was happening. I have now figured out why and I'll push a fix.